run.sh

```diff
@@ -0,0 +1 @@
+/Users/hsum/homebrew/bin/analytics --output /Users/hsum/Desktop/real_time_cfpb/analytics.usa.gov
```

Comment:
Sorry, the scripts in the front end repo were old and we're not going to
use those. The script we will use is in the private ticker_installer repo.
I deleted those scripts from the front end repo.

Dri

On Mon, Nov 16, 2015 at 4:20 PM, Dri Torres <dri.nicole.torres@gmail.com>
wrote:

> For some reason cron likes absolute paths, but I can try it in vagrant and
> see if that will work.
>
> Sent from my iPhone
>
> On Nov 16, 2015, at 3:53 PM, Tim Anderegg <notifications@github.com>
> wrote:
>
> In run.sh
> <https://github.com/cfpb/analytics.usa.gov/pull/1#discussion_r44981352>:
>
> > @@ -0,0 +1 @@
> > +/Users/hsum/homebrew/bin/analytics --output /Users/hsum/Desktop/real_time_cfpb/analytics.usa.gov
>
> OK, you should be able to just run "analytics --output etc..." without the
> path, since it is being installed as an NPM package. If you could commit
> the change to your repo when you get a chance, let me know. Thanks!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cfpb/analytics.usa.gov/pull/1/files#r44981352>.
>
>